### ISSUE TITLE
swift iOS 12 animation.adjustBlock闪退 Thread 1: "Array bound, please check it carefully."

### ISSUE THREAD

**Issue by wyxuan** [tigerAndBull/TABAnimated#120]
### New Issue Checklist

* [ ] I have read the [Documentation](https://github.com/tigerAndBull/TABAnimated/tree/master/Documents)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/TABAnimated/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS / macOS
 Platform Version        | e.g. 12.0 / 10.14.0
 TABAnimated Version      | e.g. 5.0.0 / 4.4.0
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by tigerAndBull** [comment]
index溢出了，可能是不同版本层级不同




------------------ 原始邮件 ------------------
发件人: wyxuan <notifications@github.com&gt;
发送时间: 2020年10月26日 12:00
收件人: tigerAndBull/TABAnimated <TABAnimated@noreply.github.com&gt;
抄送: Subscribed <subscribed@noreply.github.com&gt;
主题: 回复：[tigerAndBull/TABAnimated] swift iOS 12 animation.adjustBlock闪退 Thread 1: "Array bound, please check it carefully." (#120)





 
New Issue Checklist
  
 I have read the Documentation 

 I have searched for a similar issue in the project and found none 
 
Issue Info
    Info Value 
    Platform Name e.g. iOS / macOS 
  Platform Version e.g. 12.0 / 10.14.0 
  TABAnimated Version e.g. 5.0.0 / 4.4.0 
  Xcode Version e.g. Xcode 11 / Xcode 10 
  Repro rate e.g. all the time (100%) / sometimes x% / only once 
  Repro with our demo prj e.g. does it happen with our demo project? 
  Demo project link e.g. link to a demo project that highlights the issue 
   
Issue Description and Steps
 
Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by tigerAndBull** [comment]
把代码的断言去掉，然后上线。具体统一的解决方案需要你提供case
